lib/broccoli/ember-app.js

```diff
@@ -742,16 +733,16 @@ class EmberApp {
   */
   _configReplacePatterns() {
     return [{
-      match: /\{\{rootURL\}\}/g,
```

Comment:
apparently not, no. my IDE actually told me that it's not needed and it seems to work fine without the escaping.